shared/utils/graphql.ts

```diff
@@ -266,7 +266,7 @@ export function parseAssertionGraphEntity(
   } = assertion;
   return {
     chainId,
-    oracleType: "Optimistic Oracle V3",
+    oracleType: "Optimistic Oracle V3" as const,
```

Comment:
everythign below here is basically the fallout from making more props undefined, we have to check more values exist now